src/Serializing/SExpression/Precise.hs

```diff
@@ -23,30 +22,31 @@ pad n = stringUtf8 (replicate (2 * n) ' ')
 
 
 class ToSExpression t where
-  toSExpression :: t -> Int -> Builder
+  toSExpression :: t ann -> Int -> Builder
```

Comment:
`ToSExpression` is defined at `* -> *` now, allowing us to distinguish between annotations & `Text` fields.

Reply:
Kind signature on the `t` on line #24 to make this explicit?

Reply:
Done in 1ec330fbe4d70d9685450e8d2c2e704b9378d445.